ethereum/gnosis_protocol_v2/view_batches.sql

```diff
@@ -24,7 +24,7 @@ WITH batch_counts AS (
                 sum(trade_value_usd) as batch_value,
                 sum(fee_usd)         as fee_value,
                 price                as eth_price
-         from gnosis_protocol_v2."view_trades"
+         from gnosis_protocol_v2.trades
```

Comment:
I suspect this may have also been an issue with the view taking so long to generate, but this is just a hunch.

Reply:
okay, i will rebuild this batches view (until you later delete)